modules/customize-certificates-understanding-service-serving.adoc

```diff
@@ -15,5 +15,18 @@ algorithm to generate service certificates.
 The generated certificate and key are in PEM format, stored in `tls.crt`
 and `tls.key` respectively, within a created secret. The
 certificate and key are automatically replaced when they get close to
-expiration. The service CA certificate, which signs the service
-certificates, is only valid for one year after {product-title} is installed.
+expiration.
+
+The service CA certificate, which issues the service certificates, is valid for 26 months and is automatically rotated when there is less than six months validity left. After rotation, the previous service CA configuration is still trusted until its expiration. This allows a grace period for all affected services to refresh their key material before the expiration. If you do not upgrade your cluster during this grace period, which restarts services and refreshes their key material, you might need to manually restart services to avoid failures after the previous service CA expires.
+
+[NOTE]
+====
+You can use the following command to manually restart all Pods in the cluster. Be aware that this command will cause a service interruption, as it goes through and deletes every running Pod in every namespace. These Pods will automatically restart after they are deleted.
```

Comment:
I'd
s/this command will cause/running this command causes
s/as it goes through and deletes/because it deletes